src/libraries/Common/src/Interop/Windows/Kernel32/Interop.EnumProcessModulesEx.cs

```diff
@@ -9,8 +9,10 @@ internal static partial class Interop
 {
     internal static partial class Kernel32
     {
-        [LibraryImport(Libraries.Kernel32, EntryPoint = "K32EnumProcessModules",  SetLastError = true, StringMarshalling = StringMarshalling.Utf16)]
+        internal const int LIST_MODULES_ALL = 0x00000003;
+
+        [LibraryImport(Libraries.Kernel32, EntryPoint = "K32EnumProcessModulesEx",  SetLastError = true, StringMarshalling = StringMarshalling.Utf16)]
```

Comment:
Looks like the StringMarshalling can be removed
```suggestion
        [LibraryImport(Libraries.Kernel32, EntryPoint = "K32EnumProcessModulesEx",  SetLastError = true)]
```